apps/krew-website/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@apps/krew-website",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview"
+  },
+  "devDependencies": {
+    "astro": "^0.20.7",
+    "tailwindcss": "^2.1.2"
+  },
```

Comment:
As remarked in person as well;
Don't think the _app_ needs a package.json with dependencies. Those can be managed within the root of the workspace.